wiki/Shortcut_key_reference/id.md

```diff
@@ -1,167 +1,294 @@
 ---
-outdated: true
+tags:
+  - hotkey
+  - hotkeys
+  - keyboard
+  - shortcuts
+  - pintasan
 ---
 
 # Referensi Tombol Pintasan
```

Comment:
shall this leave this as lowercase to similar to original?

Reply:
true